ts/Stock/Indicators/MACD/MACDIndicator.ts

```diff
@@ -418,7 +418,8 @@ class MACDIndicator extends SMAIndicator {
                 defined(shortEMA[i + (params.shortPeriod as any) + 1][0])
             ) {
                 MACD.push([
-                    shortEMA[i + (params.shortPeriod as any) + 1][0],
+                    (params.shortPeriod as number) >= 12 ?
```

Comment:
Does not work with `shortPeriod` bigger than 12: http://jsfiddle.net/BlackLabel/30t4p9me/6/ - first points are missing in the bottom pane